tests/testthat/test-create_landscape_covariates.R

```diff
@@ -0,0 +1,12 @@
+# test_that("create_landscape_covariates fails with invalid input", {
+#
+#   outdir <- withr::local_tempdir()
+#   sp <- "C:/Programs/saga-9.2.0_x64/saga-9.2.0_x64/saga_cmd.exe"
+#   aoi_snapped <- make_test_aoi(outdir)
+#   aoi_rast <- create_template_raster(aoi_snapped, res = 50, out_dir = outdir)
```

Comment:
I think these arguments need to be updated: aoi -> dtm

Reply:
thanks - I updated these and added a placeholder for correct working function output